src/app/routes/homePage/getInitialData/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Can this and the lint rule below be removed?

Reply:
I have removed the `no-explicit-any` lint rule, but removing the other causes a problem further down the file as it's using an `@ts-expect-error` so have left that one in.

Reply:
The ban-comment lint rule is only required if there's a `ts-ignore`. Can we replace `ts-ignore` with `ts-expect-error`?